packages/circuit-ui/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -86,14 +99,15 @@ export const RadioButtonGroup = forwardRef(
       showValid,
       disabled,
       hasWarning,
+      hideLabel,
       ...props
     }: RadioButtonGroupProps,
     ref: RadioButtonGroupProps['ref'],
   ) => {
     const name = customName || uniqueId('radio-button-group_');
     return (
       <fieldset name={name} ref={ref} {...props}>
-        {label && <Legend>{label}</Legend>}
+        {label && <Legend hideLabel={hideLabel}>{label}</Legend>}
```

Comment:
I don't think we have to check. for `label` here, it's required 🙂

Reply:
In TypeScript, but not JavaScript 🙃

Reply:
Should we make this consistent?

https://github.com/sumup-oss/circuit-ui/blob/1869c21810c10182e6ace038e096c0e1d68a489c/packages/circuit-ui/components/Input/Input.tsx#L344

In the Input I assumed that _not_ passing a label would make the UI weird even if the IDE doesn't show errors, so developers would pass it (cf. what we saw in Account in the dashboard yesterday)